.github/workflows/download-image.yml

```diff
@@ -0,0 +1,59 @@
+---
+name: Docker Images
+# on:
+#   schedule:
+#     - cron: "*/15 * * * *"
+#   workflow_dispatch:
+on:
+  push:
+    paths:
+      - '.github/workflows/**'
+jobs:
+  docker-update:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      packages: write
+    env:
+      GCR_PAT: ${{ secrets.GITHUB_TOKEN }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Pull PostGREST Docker Image
+        run: docker pull postgrest/postgrest:latest
+      - name: Pull swagger Docker Image
+        run: docker pull swaggerapi/swagger-ui:latest
+      - name: Pull ClamAV Docker Image
+        run: docker pull clamav/clamav:latest
+
+      - name: grab trivy container
+        run: docker pull aquasec/trivy:latest
+      - name: Scan PostGREST
+        run: docker run  aquasec/trivy:latest image postgrest/postgrest:latest
+      - name: Scan swagger
+        run: docker run  aquasec/trivy:latest image swaggerapi/swagger-ui:latest
+      - name: Scan ClamAV
+        run: docker run  aquasec/trivy:latest image  --scanners vuln clamav/clamav:latest
+
```

Comment:
We're scanning, but pushing the images regardless of what the scan found... There should be some kind of conditional here to prevent tagging+pushing if the images have findings that we don't accept, right?

Reply:
(Note that this is a good reason to use the [Trivy GitHub action](https://github.com/marketplace/actions/aqua-security-trivy) instead of running the Docker image ourselves... We can set severity and the exit code will be used to halt/proceed, and the results are ready to include in GitHub's native facility for alerting us to problems.)